index.html

```diff
@@ -164,7 +185,8 @@ <h1>Our Sponsors</h1>
 
```

Comment:
```suggestion
    <br>
```
It might be good to add a small space between the pictures and the section name